.gitignore

```diff
@@ -1,3 +1,18 @@
+<<<<<<< HEAD
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore the default SQLite database.
+/db/*.sqlite3
+/db/*.sqlite3-journal
+/db/*.sqlite3-*
+=======
```

Comment:
this file needs to be reviewed for duplicate directives. the git merge tags need to be removed as well. (git merge tags are the lines 
```
<<<<< HEAD
```
and 
```
=======
````